core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -105,6 +107,35 @@ public void testPorts() {
     }
   }
 
+  @Test
+  public void testJson() {
+    assertFalse(TSERV_COMPACTION_SERVICE_META_EXECUTORS.getType().isValidFormat("notJson"));
+
+    // use "real" example
+    String json1 =
+        "[{'name':'small','type':'internal','maxSize':'32M','numThreads':2},{'name':'huge','type':'internal','numThreads':2}]"
+            .replaceAll("'", "\"");
+    // use synthetic, but valid json
+    String json2 =
```

Comment:
If you want to have two separate json test values, we might want to remove the top level list brackets on one of these so the test values are different forms of valid json.

Reply:
Added another sample in 7fa8acabb7 - the test requires a single json object, so removing the brackets allow fails.